source/_docs/installation.markdown

```diff
@@ -116,18 +102,12 @@ The following installs are only recommended for experienced users of those platf
   </a>
 </div>
 
-## {% linkable_title After installation %}
+### {% linkable_title After installation %}
 
 Once Home Assistant is installed, execute the following code in a console/terminal to check if the setup was successful:
```

Comment:
I think that we can drop this section. If it is useful to run `hass`, it should be part of the guides for which a command line will be available (hassbian, Debian-based, etc)